db/migrate/20190426133610_add_course_sort_by.rb

```diff
@@ -0,0 +1,5 @@
+class AddCourseSortBy < ActiveRecord::Migration[5.0]
+  def change
+    add_column :courses, :sort_by, :boolean, null: false, default: false
```

Comment:
What would this look like if we wanted to add another sorting method down the road?

Reply:
I agree, this solution does work well later down the road. I'm not sure what is the best approach. Maybe `sort_by` could be a [enum attribute](https://api.rubyonrails.org/v5.2.3/classes/ActiveRecord/Enum.html)? Then we could have a switch statement to allow for different types of sorts with a default of timestamp.

Reply:
Makes sense to me!